nixos/modules/services/misc/atuin.nix

```diff
@@ -8,6 +8,12 @@ in
     services.atuin = {
       enable = lib.mkEnableOption (mdDoc "Atuin server for shell history sync");
 
+      package = mkOption {
+        type = types.package;
+        default = [ "atuin" ];
+        description = mdDoc "Atuin package to use.";
+      };
```

Comment:
There's a `mkPackageOption` function that does something like that.

Reply:
Apologies! I was overcomplicating things. Didn't realize the `mkPackageOption` basically does everything for you.